webview/src/plots/components/comparisonTable/styles.module.scss

```diff
@@ -88,13 +88,18 @@ $gap: 4px;
   transform: rotate(0deg);
 }
 
-.missing {
+.noImage {
   background-color: $bg-color;
   border-style: solid;
   border-width: thin;
   border-color: $watermark-color;
 }
 
+.noImageContent {
+  padding-top: 25%;
+  padding-bottom: 25%;
```

Comment:
[F] Spent a couple of hours trying to find a more elegant way to do this. Please LMK if there is a better solution.

Reply:
Setting a height on the `td` would act in the same way (if other cells are bigger, the cell will get bigger as height on cells act as a min-height). I think that the only downside to your solution is that it is not intuitive why it is needed, that is until you have only cells without image content. IMO you can leave as is, you can add a comment if you want.